src/lib.rs

```diff
@@ -10,14 +10,8 @@
 // - Henry de Valence <hdevalence@hdevalence.ca>
 
 #![no_std]
-#![cfg_attr(
-    all(target_arch = "x86_64", feature = "simd_avx512", nightly),
-    feature(stdsimd)
-)]
-#![cfg_attr(
-    all(target_arch = "x86_64", feature = "simd_avx512", nightly),
-    feature(avx512_target_feature)
-)]
+#![cfg_attr(all(target_arch = "x86_64", nightly), feature(stdsimd))]
+#![cfg_attr(all(target_arch = "x86_64", nightly), feature(avx512_target_feature))]
```

Comment:
Should probably not enable these if a non-simd backend was forced.

Reply:
Yeah tru thanks for catching - How about [edeb5c](https://github.com/dalek-cryptography/curve25519-dalek/pull/531/commits/edeb5c273e0a85010956373c4af1c83974782b5a)